app/models/ems_refresh.rb

```diff
@@ -120,7 +120,9 @@ def self.get_target_objects(target, single_id = nil)
     targets_by_type = target.each_with_object(Hash.new { |h, k| h[k] = [] }) do |(target_class, id), hash|
       # Take care of both String or Class type being passed in
       target_class = target_class.to_s.constantize unless target_class.kind_of?(Class)
-      if [VmOrTemplate, Host, ExtManagementSystem, ManagerRefresh::Target].none? { |k| target_class <= k }
+
+      if ManagerRefresh::Inventory.persister_class_for(target_class).blank? &&
```

Comment:
@Ladas @agrare check this and the changes in `app/models/ems_refresh/refreshers/ems_refresher_mixin.rb` and `app/models/manager_refresh/inventory.rb`

This allows all ActiveRecord targets that have a corresponding `inventory/parser/<class_name>` to be refreshed too.